src/test/pythonnode_test.cpp

```diff
@@ -498,18 +498,228 @@ TEST_F(PythonFlowTest, PythonNodePassArgumentsToConstructor) {
     }
 }
 
+TEST_F(PythonFlowTest, PythonNodeLoopbackDefinedOnlyOnInput) {
+    ConstructorEnabledModelManager manager;
+    std::string testPbtxt = R"(
+    input_stream: "OVMS_PY_TENSOR:input"
+    output_stream: "OVMS_PY_TENSOR:output"
+        node {
+            name: "pythonNode"
+            calculator: "PythonExecutorCalculator"
+            input_side_packet: "PYTHON_NODE_RESOURCES:py"
+            input_stream: "INPUT:input"
+            input_stream: "LOOPBACK:loopback"
+            output_stream: "OUTPUT:output"
+            node_options: {
+                [type.googleapis.com / mediapipe.PythonExecutorCalculatorOptions]: {
+                    handler_path: "/ovms/src/test/mediapipe/python/scripts/symmetric_increment.py"
+                }
+            }
+        }
+    )";
+    ovms::MediapipeGraphConfig mgc{"mediaDummy", "", ""};
+    DummyMediapipeGraphDefinition mediapipeDummy("mediaDummy", mgc, testPbtxt, getPythonBackend());
+    mediapipeDummy.inputConfig = testPbtxt;
+    ASSERT_EQ(mediapipeDummy.validate(manager), StatusCode::MEDIAPIPE_GRAPH_INITIALIZATION_ERROR);
+}
+
 TEST_F(PythonFlowTest, PythonNodeLoopbackDefinedOnlyOnOutput) {
     ConstructorEnabledModelManager manager;
     std::string testPbtxt = R"(
-    input_stream: "in"
-    output_stream: "out"
+    input_stream: "OVMS_PY_TENSOR:input"
+    output_stream: "OVMS_PY_TENSOR:output"
         node {
-            name: "pythonNode2"
+            name: "pythonNode"
             calculator: "PythonExecutorCalculator"
             input_side_packet: "PYTHON_NODE_RESOURCES:py"
-            input_stream: "in"
-            output_stream: "out2"
+            input_stream: "INPUT:input"
+            output_stream: "OUTPUT:output"
+            output_stream: "LOOPBACK:loopback"
+            node_options: {
+                [type.googleapis.com / mediapipe.PythonExecutorCalculatorOptions]: {
+                    handler_path: "/ovms/src/test/mediapipe/python/scripts/symmetric_increment.py"
+                }
+            }
+        }
+    )";
+
+    ovms::MediapipeGraphConfig mgc{"mediaDummy", "", ""};
+    DummyMediapipeGraphDefinition mediapipeDummy("mediaDummy", mgc, testPbtxt, getPythonBackend());
+    mediapipeDummy.inputConfig = testPbtxt;
+    ASSERT_EQ(mediapipeDummy.validate(manager), StatusCode::MEDIAPIPE_GRAPH_INITIALIZATION_ERROR);
+}
+
+// InputStreamHandler is validated only on CalculatorNode::PrepareForRun/
+// This is called in Graph::Run/StartRun.
+// This is called only when input side packets are ready, meaning only when first request arrives.
+// With current MediaPipe implementation it is impossible to validate indexes during graph loading.
+TEST_F(PythonFlowTest, DISABLED_PythonNodeLoopback_SyncSet_WrongIndex) {
```

Comment:
DISABLED makes this code not compile. Wouldn't it be better to use GTEST_SKIP() << "Why this is skipped"?

Reply:
We could also skip just before the test would close due to error so we could potentialy discover when this is fixed in MP.

Reply:
There were 2 tests, added gtest skip for one, but for the second one there was no point - entire body would need to be skipped. I left it disabled.